Web/CSS/CSS-Assignment-1/WebContent/Profile.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+<head>
+<link rel="stylesheet" href="..\Css\Profile.css" type="text/css">
+<meta charset="ISO-8859-1">
+<title>Profile</title>
+</head>
+<body>
+	<header>
+		<h1>My Profile</h1>
+	</header>
+	<div id="maindiv">
+		<div id="firstdiv">
```

Comment:
Use id specific to content.Ex- for profile pic it should be like 'profile-pic-container' .